tracks/csharp/exercises/hamming/mentoring.md

```diff
@@ -82,6 +82,13 @@ public static int Distance(string strand1, string strand2)
 }
 ```
 
+### A single line method using lambda operator
+
+```csharp
+public static int Distance(string first, string second) =>
+        (first.Length != second.Length) ? throw new ArgumentException() : (first.Length - Enumerable.Range(0, first.Length-1).Count(index => first[index] == second[index]));
```

Comment:
I'd prefer a slightly different multiline formatting and using the same approach and variable names as the other examples:

```suggestion
        strand1.Length != strand2.Length
            ? throw new ArgumentException("Strands have different length")
            : strand1.Zip(strand2, (nucleotide1, nucleotide2) => nucleotide1 == nucleotide2 ? 0 : 1).Sum();
```